radio/apps/schedules/models.py

```diff
@@ -149,24 +145,23 @@ def __get_rrule(self):
             # Due to rrule we need to add 1 day
             end_date = end_date + datetime.timedelta(days=1)
             return rrule.rrule(
-                rrule.WEEKLY, byweekday=[self.day], dtstart=datetime.datetime.combine(
-                    start_date, self.start_hour
-                ), until=end_date
-            )
+                rrule.WEEKLY, byweekday=[self.day],
+                dtstart=datetime.datetime.combine(start_date, self.start_hour),
+                until=end_date)
         else:
             end_date = self.schedule_board.end_date
             if end_date:
                 # Due to rrule we need to add 1 day
                 end_date = end_date + datetime.timedelta(days=1)
                 return rrule.rrule(
-                    rrule.WEEKLY, byweekday=[self.day], dtstart=datetime.datetime.combine(
-                        start_date, self.start_hour
-                    ), until=end_date
+                    rrule.WEEKLY, byweekday=[self.day],
+                    dtstart=datetime.datetime.combine(start_date, self.start_hour),
+                    until=end_date
```

Comment:
Make code more readable, do not split datetime.combine function call across multiple lines resulting in spread rrule params where we are not forced to do so.